bundles/erp-order-page-search/src/FondOfOryx/Client/ErpOrderPageSearch/Plugin/SearchExtension/ToDateErpOrderPageSearchQueryExpanderPlugin.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace FondOfOryx\Client\ErpOrderPageSearch\Plugin\SearchExtension;
+
+use DateTime;
+use Elastica\Query;
+use Elastica\Query\BoolQuery;
+use Elastica\Query\Range;
+use FondOfOryx\Shared\ErpOrderPageSearch\ErpOrderPageSearchConstants;
+use Generated\Shared\Search\ErpOrderIndexMap;
+use InvalidArgumentException;
+use Spryker\Client\Kernel\AbstractPlugin;
+use Spryker\Client\SearchExtension\Dependency\Plugin\QueryExpanderPluginInterface;
+use Spryker\Client\SearchExtension\Dependency\Plugin\QueryInterface;
+
+class ToDateErpOrderPageSearchQueryExpanderPlugin extends AbstractPlugin implements QueryExpanderPluginInterface
+{
+    /**
+     * @param \Spryker\Client\SearchExtension\Dependency\Plugin\QueryInterface $searchQuery
+     * @param array $requestParameters
+     *
+     * @return \Spryker\Client\SearchExtension\Dependency\Plugin\QueryInterface
+     */
+    public function expandQuery(QueryInterface $searchQuery, array $requestParameters = [])
+    {
+        if (!isset($requestParameters[ErpOrderPageSearchConstants::PARAMETER_TO])) {
+            return $searchQuery;
+        }
+
+        $to = new DateTime($requestParameters[ErpOrderPageSearchConstants::PARAMETER_TO]);
+        $boolQuery = $this->getBoolQuery($searchQuery->getSearchQuery());
+
+        $toRange = (new Range())->addField(
+            ErpOrderIndexMap::CREATED_AT,
+            [
+                'lte' => $to->format('Y-m-d H:i:s'),
```

Comment:
Maybe a constant for the format will be better. You can use this constants also for the class `FromDateErpOrderPageSearchQueryExpanderPlugin`.